packages/components/stories/Notification.js

```diff
@@ -27,32 +26,38 @@ class NotificationWrapper extends React.Component {
 					},
 				},
 			]);
-			this.setState({	counter: this.state.counter + 1 });
+			this.setState(prevState => ({ counter: prevState.counter + 1 }));
```

Comment:
i don't see the point of doing that here since what the function is doing here is really simple.